Source/WebCore/bindings/js/ScheduledAction.cpp

```diff
@@ -94,7 +94,6 @@ void ScheduledAction::executeFunctionInContext(JSGlobalObject* globalObject, JSV
     ASSERT(m_function);
     VM& vm = context.vm();
     JSLockHolder lock(vm);
-    auto catchScope = DECLARE_CATCH_SCOPE(vm);
```

Comment:
This still serves a purpose to detect if there are other uncaught exceptions.  I think we should leave it in.  It costs almost nothing for Release builds.